src/orcania.c

```diff
@@ -1,9 +1,11 @@
-#define _GNU_SOURCE
+#ifndef _GNU_SOURCE
+# define _GNU_SOURCE
```

Comment:
Why do you define `_GNU_SOURCE` like this? I'm not sure I get it.

Reply:
To avoid conflict with the building when it already enabled this flags, please see: [silvioprog/orcania/blob/cmake/CMakeLists.txt#L30](https://github.com/silvioprog/orcania/blob/cmake/CMakeLists.txt#L30).